### ISSUE TITLE
Can you offer example data?

### ISSUE THREAD

**Issue by jhonsonlaid** [richard-roberts/PhD#1]

**Comment by richard-roberts** [comment]
Yes, there meant to be a bunch of results and data on the [website](https://richard-roberts.github.io/SalientPoses/) (but for some reason the links are broken, I'll look into it another time).

For now, check out the [data folder](https://github.com/richard-roberts/SalientPoses/tree/master/data). That has both CSV and Maya scene files. The original FBX files can be found [here](https://github.com/richard-roberts/SalientPosesPerformance/tree/master/expected/original).

Let me know how you get on :)
